requirements-test.txt

```diff
@@ -1,15 +1,16 @@
+# Tools for static checking.
 black == 19.10b0
 flake8 == 3.7.9
 isort == 5.6.4
-mpi4py == 3.0.3
-mypy == 0.770
+mypy == 0.790
```

Comment:
does that change anything, practically ?

Reply:
only mypy version is updated so that py39 can work. no impact on py37 and py38.

Reply:
mpi4py is moved into a group only.